mono/io-layer/handles.c

```diff
@@ -50,6 +50,33 @@
 #include <mono/utils/mono-logger-internals.h>
 #undef DEBUG_REFS
 
+struct _WapiHandleUnshared
+{
```

Comment:
If we move things around, it would be nice to clean up the naming of these structures, i.e. drop the underscores. Also this could be named simply WapiHandle.